### PR TITLE
Expert Progression continuation

### PR DESCRIPTION
**Pull request by MuteTiefling**
## Misc
Disable all power generation, except Powah Reactors (late game) and Enigmatic Unity (Early/Mid Game)

Simplify the various jars a bit

Thermal Jar as 'unified' base material: 
image
Supplementaries and Thermal are just a simple swap
image

Hexerei Herb Jars
image

Source Jars
image
Containment Jars
image
Fluid Containement Jars
image

Jar of Light made accessible early
image
And Jar of Voiding
image

Immersive Engineering Water Wheels and Windmills
Without the dynamo to make these generate power, they are pretty much just decorational. Simplified recipes
image
image

Time in a Bottle
image

Nomadic Tents
Re-working the materials to ensure they're all available through early game Twilight Forest
Tent Canvas
image

Basic Hammer
image

Golden Hammer
image

Crossbeams for higher tier tents
image
image
image

Indlu Wall
Straw, since shearing leaves is tough
image

Shovels
These add depth to the inside of a tent. 
image
image
image
image
image

Bringing forward a bunch of NA utility items to various points in the Twilight Forest
Bucket of Infinite Color
image

NA Staves
image
image
image

image
image
image

Water Spring
image

Field Creator
image

### Helper Recipes for Potions
Getting dragons breath for lingering potions for the first Aura gen can be done more easily once the player has unlocked Blue Skies and Bumblezone via this recipe

Until then, they've got this.
Blood may be extracted the normal way with Hexerei (manual, requires jumping on spikes) or automated with this recipe that needs some raw meat and bug guts/spider eyes.
image
Blood can be bottled:
image
Converted to Fiery Blood
image
And finally Dragon's Breath
image

## Obtaining Spells
Tier 1 spell glyphs may be obtained from Death Tomes. To facilitate this, they can be summoned after some progress
image
image
Charms of life may be found all over twilight, but one is guaranteed from the lich. Imbuement doesn't consume the key items, so this can just remain set up and running.

### CODE REVIEW

**Comment by MuteTiefling** [comment]
### Tech - Pneumaticcraft
Move TPP into Compressed Stone tier
image
Add Creosote/charcoal to TPP for access to Treated Wood
image
As well as a TPP recipe to make the Treated Wood
image

**Comment by MuteTiefling** [comment]
### Magic

**Comment by MuteTiefling** [comment]
Alright, should be good on this. This is Chapter 1 basically done, with Chapter 2 beginning with the player being sent to Blue Skies.

File: kubejs/server_scripts/force_reload.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ServerEvents.loaded((event) => {
+    event.server.runCommandSilent('reload');
```
Comment (NielsPilgaard) [inline kubejs/server_scripts/force_reload.js:2]: what's going on here? 😄

**Review by NielsPilgaard — APPROVED** [review]

File: kubejs/server_scripts/force_reload.js (hunk 1 above)
Comment (MuteTiefling) [reply kubejs/server_scripts/force_reload.js:2]: That's the 'fix' for tags blowing up. Just one quick reload as the server starts. It's been almost imperceptible so far

Comment (NielsPilgaard) [reply kubejs/server_scripts/force_reload.js:2]: Cool 👍

**Pull request merged**